CHANGES.md

```diff
@@ -1,3 +1,254 @@
+
+* [#724](https://github.com/typelevel/cats/pull/724): sbt-doctest is now used to
+  validate Scaladoc examples
+* [#809](https://github.com/typelevel/cats/pull/809): MiMa now enforces binary
+  compatibilty
+* [#806](https://github.com/typelevel/cats/pull/806): Various improvements to
+  use of Simulacrum, which is now a compile-time-only dependency
+* [#734](https://github.com/typelevel/cats/pull/734): Documentation on testing
+  conventions
+* [#710](https://github.com/typelevel/cats/pull/710): Documentation for
+  `Invariant`
+* [#832](https://github.com/typelevel/cats/pull/832): Updated `Free`
+  documentation
+* [#824](https://github.com/typelevel/cats/pull/824): New examples for
+  `Foldable`
+* [#797](https://github.com/typelevel/cats/pull/797): Scaladoc examples for
+  methods on `Arrow`
+* [#783](https://github.com/typelevel/cats/pull/783) and others: Scaladoc
+  examples for syntax methods
+* [#720](https://github.com/typelevel/cats/pull/720): Expanded documentation for
+  `FreeApplicative`
+* [#636](https://github.com/typelevel/cats/pull/636): Law checking for `Eval`
+* [#649](https://github.com/typelevel/cats/pull/649) and
+  [#660](https://github.com/typelevel/cats/pull/660): Better `Arbitrary`
+  instances for `Streaming` and `StreamingT`
+* [#722](https://github.com/typelevel/cats/pull/722): More consistent `toString`
+  for `StreamingT`
+* [#672](https://github.com/typelevel/cats/pull/672): Additional laws for
+  `Profunctor`
+* [#668](https://github.com/typelevel/cats/pull/668),
+  [#669](https://github.com/typelevel/cats/pull/669),
+  [#679](https://github.com/typelevel/cats/pull/679),
+  [#680](https://github.com/typelevel/cats/pull/680), and
+  [#681](https://github.com/typelevel/cats/pull/681): Additional law checking
+  for `Xor`, `XorT`, and `Either`
+* [#707](https://github.com/typelevel/cats/pull/707): Additional testing for
+  `State` and `StateT`
+* [#736](https://github.com/typelevel/cats/pull/736): `map` / `flatMap`
+  coherence
+* [#748](https://github.com/typelevel/cats/pull/748): Left and right identity
+  laws for `Kleisli`
+* [#753](https://github.com/typelevel/cats/pull/753): Consistency tests for
+  `Cokleisli`
+* [#733](https://github.com/typelevel/cats/pull/733): Associativity laws for
+  `Kleisli` and `Cokleisli` composition
+* [#741](https://github.com/typelevel/cats/pull/741): Tests for
+  `Unapply`-supported syntax
+* [#690](https://github.com/typelevel/cats/pull/690): Error reporting
+  improvements for serializability tests
+* [#701](https://github.com/typelevel/cats/pull/701): Better documentation for
+  the Travis CI script
+* [#787](https://github.com/typelevel/cats/pull/787): Support for cross-module
+  Scaladoc links
+
+Known issues:
+
+* [#702](https://github.com/typelevel/cats/pull/702): This change identified and
+  fixed a stack safety bug in `foldMap` on `Free`, but raised other issues (see
+  [#712](https://github.com/typelevel/cats/issues/712)) and was reverted in
```

Comment:
@refried What do you mean? The first closes the link and the second the parenthetical.